utils/build.gradle

```diff
@@ -4,7 +4,4 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
     implementation 'androidx.core:core-ktx:1.3.1'
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
```

Comment:
There are no tests on the `utils` module so these are unneeded

Reply:
Same for these, I also noticed them when I was trying to make the compilation faster. Do you mind opening a PR?

Reply:
Or create a task in Jira, it's not a high priority 😄